tests/test_formatting.py

```diff
@@ -316,6 +316,8 @@ def cmd1():
 
 
 def test_global_show_default(runner):
+    """Note: default to "--help" would not be shown"""
```

Comment:
Add this as a comment, docstrings are meant to describe a test not add comments.

Reply:
alright